README.md

```diff
@@ -68,6 +72,67 @@ When you pass down props to the web component, instead of setting attributes lik
 <MyComponent items={elem.items} callback={elem.callback} />
 ```
 
+### Reactifying web components loaded using HTML import
+Custom elements that depend on HTML imports (which are described in another part of the Web Components specification) were previously a bit trickier to integrate into a React project. But this `react-integration` library combined with [the `web-components` Webpack loader](https://github.com/rnicholus/web-components-loader) make the process farily painless. After integrating the Webpack loader into your project, simply `import` the root HTML file of the web component in your React component and use the generated URL to import the Web component using an HTML import in your `render` method. For example:
+
+```jsx
+import React, { Component } from 'react'
+
+import reactify from 'skatejs-react-integration'
+
+const importWcUrl = require('my-web-component/component.html')
+
+class MyWebComponentWrapper extends Component {
+  render() {
+    const MyComponent = reactify('my-web-component')
+
+    return (
+      <span>
+          <link rel='import' href={ importWcUrl } />
+          <MyComponent />
+      </span>
+    )
+  }
+}
+
+export default MyWebComponentWrapper
+```
+
+### Web component attributes
+If the underlying web component you intend to Reactify requires some properties to be set directly on the element as attributes, include an `attr-` prefix on the property name. For example:
+
+```jsx
+<MyComponent attr-data-fo='bar' />
+```
+
+The above code will set a `data-foo` attribute on the underlying custom element, instead of setting a corresponding property on the element object. An example of such a web component is [the hugely popular `<x-gif>` element](https://github.com/geelen/x-gif), which requires the GIF `src` to be set as an element and _not_ a property.
+
+### Type-extension elements
+Custom elements that extend an existing native element are also supported. Take [the ajax-form element](https://github.com/rnicholus/ajax-form) as an example. Ajax-form extends the native `<form>` element to provide additional features. This is an example of a type-extension element. In order to use any type-extension element, such as ajax-form, your render method might look something like this:
+
+```jsx
+render() {
+  const Form = reactify('form')
```

Comment:
I commented in the particular parts of the code that could handle this. Essentially, you'd try and look up the component with `customElements.get(name)` and then construct it using the  constructor. Underneath the hood you could tell React to use the real `tagName` and set the `is` attribute.

I'm unsure how this will work, though. The v1 CE polyfill doesn't support customised built-ins, I don't think. I also don't think React will call `createElement()` with the correct arguments in order for them to work, even if it did (or the browser natively supported them).